src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java

```diff
@@ -676,7 +677,7 @@ void setHearingContactDate(Map<String, Object> personalisation, NotificationSscs
 
     Map<String, Object> setEventData(Map<String, Object> personalisation, SscsCaseData ccdResponse, NotificationEventType notificationEventType) {
         if (ccdResponse.getCreatedInGapsFrom() != null && ccdResponse.getCreatedInGapsFrom().equals("readyToList")) {
-            LocalDate localDate = LocalDate.parse(ofNullable(ccdResponse.getDateSentToDwp()).orElse(LocalDate.now().toString())).plusDays(calculateMaxDwpResponseDays(ccdResponse.getBenefitCode()));
+            LocalDate localDate = LocalDate.parse(ofNullable(ccdResponse.getDateSentToDwp()).orElse(LocalDate.now().toString())).plusDays(calculateMaxDwpResponseDays(ccdResponse.getAppeal().getBenefitType()));
```

Comment:
Maybe you can't answer this but is the BenefitType always present within the Appeal obj?